server/src/main/java/org/elasticsearch/index/query/PrefixQueryBuilder.java

```diff
@@ -137,8 +136,8 @@ public void doXContent(XContentBuilder builder, Params params) throws IOExceptio
         if (rewrite != null) {
             builder.field(REWRITE_FIELD.getPreferredName(), rewrite);
         }
-        if (caseInsensitive != DEFAULT_CASE_INSENSITIVITY) {
-            builder.field(CASE_INSENSITIVE_FIELD.getPreferredName(), caseInsensitive);
+        if (caseSensitivityMode != CaseSensitivityMode.FIELD_DEFAULT) {
+            builder.field(CaseSensitivityMode.KEY.getPreferredName(), caseSensitivityMode.parseField().getPreferredName().toString());
```

Comment:
Seems that in  `getPreferredName().toString()` ,`toString()` part  is redundant.  
Here and in other queryBuilers as well